### PR TITLE
Feature/logging and kryptnostic services

### PR DESCRIPTION
**Pull request by juliannaelamb**
Added tracking for document sizes.

### CODE REVIEW

**Comment by juliannaelamb** [comment]
@nickdhewitt @makosblade @geekbeast

File: src/main/java/com/kryptnostic/kodex/v1/constants/Names.java [hunk 1]
```diff
@@ -70,4 +70,5 @@ private Names() {}
     public static final String CHILD_OBJECT_COUNT_FIELD      = "childObjectCount";
     public static final String RESET_TOKEN_FIELD             = "resetToken";
     public static final String REGISTRAR_USER_SUFFIX         = "_registrar";
+	public static final String SIZE_FIELD 					 = "size";
```
Comment (geekbeast) [inline src/main/java/com/kryptnostic/kodex/v1/constants/Names.java]: ?

**Comment by juliannaelamb** [comment]
So weird, the formatting showed up fine in eclipse not sure what was going on there but should be good now!

**Comment by geekbeast** [comment]
Closing for now, changes not needed.

**Comment by juliannaelamb** [comment]
Don't we still need the document size info in ObjectMetadata?

**Comment by juliannaelamb** [comment]
@geekbeast bumping this, I think some of the changes are still needed to add the size field to ObjectMetadata.

**Pull request closed without merging**